src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -151,13 +151,40 @@
   (reset! filtered-data nil)
   (reset! search-text ""))
 
+(defn f-view
+  [{:keys [render-emojis? search-text on-change-text clear-states active-category scroll-ref theme]
+    :as   sheet-opts}]
+  (let [search-active? (pos? (count @search-text))]
+    (rn/use-effect #(js/setTimeout (fn [] (reset! render-emojis? true)) 250))
```

Comment:
This line is the only change. The reset is refactoring.

Reply:
how does it work? what was causing the issue?

Reply:
@J-Son89 rendering the emojis data is heavy on the UI thread, we meed to delay rendering until the navigation animation completes

Reply:
Thanks Omar, maybe we should comment in the file about this?
Should the timeout time be a named def to help give context?

Reply:
> Thanks Omar, maybe we should comment in the file about this?

@J-Son89 added comment. 

> Should the timeout time be a named def to help give context?

No need I think, it is used only once, and the ratom is named `render-emojis?` inside a `setTimeout` so I think it is clear what is happening